File: java/src/main/java/io/temporal/samples/moneytransfer/model/DepositResponse.java
package io.temporal.samples.moneytransfer.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DepositResponse {

    @JsonProperty("chargeId")
    private String depositId;
}

File: dotnet/src/model/DepositResponse.cs
using System.Text.Json.Serialization;

namespace MoneyTransfer;

public class DepositResponse
{
    [JsonPropertyName("chargeId")]
    public string DepositId { get; set; }

    public DepositResponse()
    {
        DepositId = "";
    }

    public DepositResponse(string id)
    {
        DepositId = id;
    }
}
